big_tests/tests/metrics_api_SUITE.erl

```diff
@@ -247,9 +247,11 @@ cluster_size(Config) ->
       ?assert_equal(1, SingleNodeClusterState2).
 
 tcp_connections_increases_with_user_connection(Config) ->
+    rpc(mim(), exometer, repair, [[global, tcpPortsUsed]]),
```

Comment:
Why do we need these?

Reply:
To force exometer to launch the sample function again, otherwise exometer waits for the timeout.